docs/operators/filter.md

```diff
@@ -0,0 +1,49 @@
+Filter
+=============
+
+## Operator Objective
+This operator receives an POJO ([Plain Old Java Object](https://en.wikipedia.org/wiki/Plain_Old_Java_Object)) as an incoming tuple
+and based on the filter condition it shall emit the filtered POJO on one output port and the rest on another output port.
+
+Filter operator supports quasi Java expressions to specify filter rule.
```

Comment:
Might not be for this review. But we need a documentation on quasi-Java expressions.